pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,19 +18,18 @@
 
 
 def sort_file_contents(f):
-    before = tuple(f)
+    before = [line.strip(b'\n\r') for line in f if line.strip()]
```

Comment:
`requirements-txt-sorter` handles this by grouping comment / blank lines for each "meaty" line and preserving them through sorting.  straight `sort` (while simpler) will put all of the comments / blank lines together (at the top? (not at a terminal to check))